jablib/src/main/resources/l10n/JabRef_en.properties

```diff
@@ -2953,3 +2908,28 @@ You\ must\ select\ an\ identifier\ type.=You must select an identifier type.
 You\ must\ specify\ a\ Bib(La)TeX\ source.=You must specify a Bib(La)TeX source.
 You\ must\ specify\ an\ identifier.=You must specify an identifier.
 You\ must\ specify\ one\ (or\ more)\ citations.=You must specify one (or more) citations.
+
+# CommandLine
+Available\ export\ formats\:=Available export formats:
+Available\ import\ formats\:=Available import formats:
+Cannot\ embed\ metadata\ on\ any\ linked\ files\ of\ %s.\ Make\ sure\ there\ is\ at\ least\ one\ linked\ file\ and\ the\ path\ is\ correct.%n=Cannot embed metadata on any linked files of %s. Make sure there is at least one linked file and the path is correct.%n
+Cannot\ write\ XMP\ metadata\ on\ any\ linked\ files\ of\ %0.\ Make\ sure\ there\ is\ at\ least\ one\ linked\ file\ and\ the\ path\ is\ correct.%n=Cannot write XMP metadata on any linked files of %0. Make sure there is at least one linked file and the path is correct.%n
+Checking\ consistency\ of\ '%0'.=Checking consistency of '%0'.
+Checking\ integrity\ of\ '%0'.=Checking integrity of '%0'.
+Converting\ '%0'\ to\ '%1'=Converting '%0' to '%1'
+Created\ library\ with\ '%0'\ entries.=Created library with '%0' entries.
+Creating\ excerpt\ of\ from\ '%0'\ with\ '%1'=Creating excerpt of from '%0' with '%1'
+Exporting\ '%0'=Exporting '%0'
+File\ %0\ is\ not\ linked\ to\ any\ entry\ in\ database.=File %0 is not linked to any entry in database.
+Import\ preferences\ from\ file=Import preferences from file
+Input\ file\ '%0'\ is\ invalid\ and\ could\ not\ be\ parsed.=Input file '%0' is invalid and could not be parsed.
+No\ library\ generated.=No library generated.
+Regenerating\ citation\ keys\ according\ to\ metadata.=Regenerating citation keys according to metadata.
+Successfully\ embedded\ XMP\ metadata\ of\ at\ least\ one\ entry\ to\ %0.=Successfully embedded XMP metadata of at least one entry to %0.
+Successfully\ embedded\ metadata\ on\ at\ least\ one\ linked\ file\ of\ %0.=Successfully embedded metadata on at least one linked file of %0.
+Successfully\ written\ XMP\ metadata\ of\ at\ least\ one\ entry\ to\ %0.=Successfully written XMP metadata of at least one entry to %0.
+Successfully\ written\ XMP\ metadata\ on\ at\ least\ one\ linked\ file\ of\ %0.=Successfully written XMP metadata on at least one linked file of %0.
+The\ following\ providers\ are\ available\:=The following providers are available:
```

Comment:
Line 2913 says `Available export formats:` - shouldn't it be `Available providers:` here?